synthetics_AWS/terraform/ec2.tf

```diff
@@ -0,0 +1,38 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
```

Comment:
Note that AMIs get deprecated and unavailable with time, this may deserve at least a comment